### PR TITLE
Retrieve tx schema - Closes #3432

### LINKED ISSUE #3432: Retrieve transaction schemas from lisk-service
### Expected behavior
Transaction schemas should be retrieved from lisk-service. 

### Actual behavior
Schemas are hard coded in the repo.

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?
This PR resolves #3432

### How was it solved?
- Get schema from API
- Store schemas in `moduleAssets.js`

### How was it tested?
Add unit test

### CODE REVIEW

File: src/app/index.js [hunk 1]
```diff
@@ -33,6 +37,12 @@ const App = ({ history }) => {
     dispatch(watchListRetrieved());
   }, []);
 
+  useEffect(() => {
+    Object.values(MODULE_ASSETS_NAME_ID_MAP).forEach((id) => {
+      selectSchema(id, network);
```
Comment (UsamaHameed) [inline src/app/index.js:42]: Please rename to something more appropriate like `retrieveSchemas` and do `Object.values(...)...` inside that function.

File: src/app/index.js [hunk 2]
```diff
@@ -10,19 +10,23 @@ import './variables.css';
...
 import ThemeContext from '../contexts/theme';
 import styles from './app.css';
 import useIpc from '../hooks/useIpc';
 
+// eslint-disable-next-line max-statements
 const App = ({ history }) => {
   const dispatch = useDispatch();
   const [loaded, setLoaded] = useState(false);
   const theme = useSelector(state => (state.settings.darkMode ? 'dark' : 'light'));
+  const network = useSelector(state => state.network);
```
Comment (UsamaHameed) [inline src/app/index.js:28]: Please add a selector for service url instead. `useSelector(selectServiceUrl)`

File: src/utils/api/transaction/lsk.js [hunk 3]
```diff
@@ -274,16 +275,16 @@ export const create = ({
...
-  console.log('create', moduleAssetType);
-  const transaction = createTransactionObject(rawTransaction, moduleAssetType);
-
   try {
-    const signedTransaction = transactions.signTransaction(
-      schema, transaction, networkIdentifier, passphrase,
-    );
-
-    resolve(signedTransaction);
+    selectSchema(moduleAssetType, network)
```
Comment (UsamaHameed) [inline src/utils/api/transaction/lsk.js:279]: This function should stay the same.

File: src/utils/api/transaction/lsk.js [hunk 4]
```diff
@@ -366,16 +367,16 @@ export const getTransactionBaseFees = network =>
...
 }) => {
   const numberOfSignatures = DEFAULT_NUMBER_OF_SIGNATURES;
   const feePerByte = selectedPriority.value;
   const {
     moduleAssetType, ...rawTransaction
   } = transaction;
 
-  const schema = selectSchema(moduleAssetType);
-  const maxAssetFee = MAX_ASSET_FEE[moduleAssetType];
+  const schema = await selectSchema(moduleAssetType, network);
```
Comment (UsamaHameed) [inline src/utils/api/transaction/lsk.js:378]: This function should stay the same.

**Review by UsamaHameed — CHANGES_REQUESTED** [review]

File: src/utils/moduleAssets.js [hunk 5]
```diff
@@ -19,7 +20,12 @@ const getModuleAssetTitle = (t = str => str) => ({
 });
 
 
-const selectSchema = moduleAssetType => MODULE_ASSETS_MAP[moduleAssetType].schema;
+const retrieveSchemas = (network) => {
+  Object.values(MODULE_ASSETS_NAME_ID_MAP).forEach(async (moduleAssetId) => {
```
Comment (UsamaHameed) [inline src/utils/moduleAssets.js:24]: I don't think we need to make several requests. If I remember correctly, you can make the request without any parameters and it should return all the schemas (which is fine for our case).

**Review by UsamaHameed — CHANGES_REQUESTED** [review]

**Review by UsamaHameed — APPROVED** [review]

**Review by reyraa — APPROVED** [review]

**Pull request merged**